firmware/controllers/flash_main.cpp

```diff
@@ -97,7 +107,7 @@ bool getNeedToWriteConfiguration() {
 }
 
 void writeToFlashIfPending() {
-#if EFI_FLASH_WRITE_THREAD
+#if (EFI_FLASH_WRITE_THREAD == TRUE)
```

Comment:
that's unusual do we really need? what's special specifically here?

Reply:
Just a style alignment to ChibiOS.